receiver/m365receiver/logs.go

```diff
@@ -246,8 +246,17 @@ func (l *m365LogsReceiver) transformLogs(now pcommon.Timestamp, audit *auditMeta
 	for _, log := range data {
 		logRecord := scopeLogs.LogRecords().AppendEmpty()
 
-		// log body
-		logRecord.Body().SetStr(log.body)
+		// parses body string and sets that as log body, but uses string if parsing fails
+		parsedBody := map[string]interface{}{}
```

Comment:
nit: `any` is an alias for `interface{}`, you'll see `interface{}` in a lot of old code. We should use `any` in new code we write.
```suggestion
		parsedBody := map[string]any{}
```